docs: add english document (#17)
ecomfe/veui-docs: one/docs/en-US/components/badge.md

# Badge

> ## Demo

Reviewer: ```suggestion
## Demos
```

### Text badge

Badge content can be text.

[[ demo src="/demo/badge/text.vue" ]]

> ### Digital badge

Reviewer: ```suggestion
### Numeric badge
```

> Badge content can be number, When the maximum value is exceeded, it can be displayed as *Maximum+*.

Reviewer: ```suggestion
The content can also be a number and can be can be displayed as “<var>max</var>+” when exceeding the maximum value.
```

[[ demo src="/demo/badge/number.vue" ]]

> ### Round Dots badge

Reviewer: 这里好像原文写错了，应该是飘角徽标，这里写 Corner badge 吧

> No badge content is provided, The badge will be displayed in a dot style in the upper right corner.

Reviewer: ```suggestion
When no badge content is provided, the badge will be displayed in the upper right corner in a dot style.
```

[[ demo src="/demo/badge/corner-dot.vue" ]]

> 

Reviewer: 少了个标题，Dot badge 吧

No slot content is provided, the badge will be displayed in a dot style.

[[ demo src="/demo/badge/dot.vue" ]]

## API

### Props

| Name | Type | Default | Description |
| -- | -- | -- | -- |
| ``type`` | `string=` | `'error'` | [^type] |

> | ``value`` | `string | number` | - | Badge content value. When it is a value of type `number`, it is limited by the [`max`](#props-max) attribute. `max` is ignored when it is of type `string`. |

Reviewer: ```suggestion
| ``value`` | `string | number` | - | The badge content value. When it is of type `number`, it is limited by the [`max`](#props-max) prop. `max` is ignored when it is of type `string`. |
```

> | ``max`` | `number=` | `badge.max` | The maximum value of the badge value, when `value` exceeds this value, the badge content will be displayed as `{max}+`. Ignored when `value` is `string`. |

Reviewer: ```suggestion
| ``max`` | `number=` | `badge.max` | The maximum value of the badge, when `value` exceeds this value, the badge content will be displayed as “<var>max</var>+”. Ignored when `value` is `string`. |
```

> | ``hidden`` | `boolean=` | `false` | Whether hidden state. |

Reviewer: ```suggestion
| ``hidden`` | `boolean=` | `false` | Whether the badge is hidden. |
```

^^^type

> Badge state type. The optional values for the built-in style are as follows. When using other values, you need to define the style of `.veui-badge-{type}`.

Reviewer: ```suggestion
The status type of the badge. The optional values for built-in styles are as follows. When using other values, you need to define the styles for `.veui-badge-{type}`.
```

+++Enum
| Value | Description |
| -- | -- |
| `info` | Info style. |
| `success` | Success style. |
| `warning` | Warning style. |
| `error` | Error style. |
| `aux` | Aux style. |
+++
^^^

### Slots

| Name | Description |
| -- | -- |

> | ``default`` | The target content that the badge needs to display. |

Reviewer: ```suggestion
| ``default`` | The target content that the badge needs to display upon. |
```

> 

Reviewer: ```suggestion
```
